python/fbprophet/forecaster.py

```diff
@@ -33,6 +35,9 @@
     plot_yearly,
     plot_seasonality,
 )
+# import fbprophet.hdays as hdays
+import hdays as hdays
+import holidays
```

Comment:
It doesn't look like `holidays` package needs to be imported here since it is imported in `hdays`. If we do need to import it, then it should be under a different name since we have a variable below named `holidays`.